app/services/supplementary-billing/unflag-unbilled-licences.service.js

```diff
@@ -27,12 +27,10 @@ const LicenceModel = require('../../models/water/licence.model.js')
  * do this because we know this service has handled anything that was unbilled and not represented.
  *
  * @param {*} billingBatchId The ID of the bill run (billing batch) being processed
- * @param {module:ChargeVersionModel[]} chargeVersions All charge versions being processed in the bill run
+ * @param {Array} allLicenceIds All licence IDs being processed in the bill run
```

Comment:
I think we've settled on using the `[]` syntax now eg. `{String[]}` for an array of strings.